lang/main-nl.json

```diff
@@ -248,7 +248,7 @@
         "stopRecording": "Opname stoppen",
         "stopRecordingWarning": "Weet u zeker dat u de opname wilt stoppen?",
         "stopStreamingWarning": "Weet u zeker dat u de livestream wilt stoppen?",
-        "streamKey": "Sleutel voor livestream",
+        "streamKey": "Livestream sleutel",
```

Comment:
Deze samentrekking hoort of aan elkaar, of met een koppelstreepje ertussen, maar persoonlijk vind ik "Sleutel voor livestream" duidelijker.

Reply:
Fair enough. I'll change it back...